code/modules/clothing/glasses/_glasses.dm

```diff
@@ -679,6 +679,7 @@
 	inhand_icon_state = "glasses"
 	glass_colour_type = /datum/client_colour/glass_colour/nightmare
 	forced_glass_color = TRUE
+	lighting_cutoff = LIGHTING_CUTOFF_FULLBRIGHT
```

Comment:
are you sure you want fullbright? we use it almost nowhere cause it ends up looking really flat. 
do you have before/after photos?

Reply:
nvm

Reply:
well it should be a nightmare